src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsOnSpecialOasis.java

```diff
@@ -0,0 +1,79 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.templates.TemplateNoFirstLoad;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsBaseObject;
+
+import org.openqa.selenium.Dimension;
+import org.testng.annotations.Test;
+
+/**
+ * https://www.google.com/dfp/5441#delivery/LineItemDetail/lineItemId=126608052
+ */
+
+public class TestAdsOnSpecialOasis extends TemplateNoFirstLoad {
+
+  @Test(groups = "TestAdsOnSpecialPagesOasis")
+  public void testAdsOnSpecialVideoPageOasis() throws Exception {
+    AdsBaseObject ads = buildAdsObjectForPage("Special:Videos");
+    verifySlotHasCorrectAd(
+        ads,
+        "TOP_LEADERBOARD",
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "TOP_LEADERBOARD" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)